lib/features/announcement/ui/announcements_list/announcements_list_page.dart

```diff
@@ -30,15 +32,34 @@ class AnnouncementsListPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      drawer: const VocelNavigationDrawer(),
       appBar: AppBar(
+        leading: Builder(
+          builder: (BuildContext context) {
+            return IconButton(
+              icon: const Icon(Icons.menu),
+              onPressed: () { Scaffold.of(context).openDrawer(); },
+              tooltip: MaterialLocalizations.of(context).openAppDrawerTooltip,
+            );
+          },
+        ),
         centerTitle: true,
-        title: const Text(
-          'Vocel Mobile App',
+        title: Text(
+          AppLocalizations.of(context)!.vocelMobileApp
         ),
         backgroundColor: const Color(constants.primaryColorDark),
       ),
       floatingActionButton: FloatingActionButton(
-        onPressed: () {},
+        onPressed: () async {
```

Comment:
Cool code - but what is the DatePicker for?

Reply:
I will move the DatePicker to the calendar page, and it might be used to pick dates and schedule meetings and events. I have added the calendar page on the home page.